### ISSUE TITLE
Drop an octave - convolution taking into account low and high frequency

### ISSUE THREAD

**Issue by agirbau** [AlexeyAB/darknet#4349]
Hello!

I was wondering if the paper [Drop an Octave: Reducing Spatial Redundancy in
Convolutional Neural Networks with Octave Convolution](https://arxiv.org/pdf/1904.05049.pdf) could be implemented. 

The basic idea is to define a convolution layer as a stack of 2 tensors: one representing the lower frequency filters and the other one representing the higher ones. 

The lower frequency filters are downsampled to half in order to have less memory consumption and higher speed. In the paper they also present an increase in accuracy.

![image](https://user-images.githubusercontent.com/23292221/69340699-5925c100-0c68-11ea-88cf-322df42e474e.png)

So I think it would be a nice addition to this amazing repo.

**Comment by agirbau** [comment]
Some implementations in tensorflow and pytorch (I haven't tried them though):
https://github.com/osmr/imgclsmob
https://github.com/lxtGH/OctaveConv_pytorch
